draft-ietf-tls-tls13.md

```diff
@@ -1566,7 +1566,9 @@ cipher_suites
   order of client preference. If the list contains cipher suites
   the server does not recognize, support, or wish to use, the server
   MUST ignore those cipher suites, and process the remaining ones as
-  usual. Values are defined in {{cipher-suites}}.
+  usual. Values are defined in {{cipher-suites}}. If the client is
+  attempting a PSK key establishment, it MUST only advertise the single
+  cipher suite associated with the PSK.
```

Comment:
@ekr: are you saying that the server should understand that the very first cipher suite, in the client hello's cipher suites list, is the one to be used with the PSK? And that if the server does not support that first choice, it should then dismiss the PSK and continue with whatever other choice of cipher suite exist? I do not see this in the spec.